pkg/volume/csimigration/plugin_manager.go

```diff
@@ -19,7 +19,6 @@ package csimigration
 import (
 	"errors"
 	"fmt"
-
```

Comment:
please leave the empty line as-is. (applies elsewhere)

Reply:
I don't see this empty line in code anymore but it still shows up here...

Reply:
@Huang-Wei asked to return it back so there is an empty line to keep separation of import blocks - stdlib vs. other

Reply:
oh, sorry, I didn't read it right the first time. Fixed